lib/facter/self_service.rb

```diff
@@ -113,6 +113,13 @@
     { S0021: Facter.value(:memory)['system']['capacity'].to_f <= 90 }
   end
 
+  chunk(:S0027) do
+    next unless PuppetSelfService.primary?
+    # Check if thundering herd is occuring.
+    psqldata = PuppetSelfService.psql_thundering_herd
+    { S0027: psqldata.count }
```

Comment:
The indicator needs to return true or false, i believe this just returns a number? We need to have it "true" if there is a thundering herd, and false if there is no thundering herd